.buildkite/pipeline.artifacts.yml

```diff
@@ -0,0 +1,13 @@
+steps:
+  - command: .buildkite/scripts/steps/artifacts/build.sh
+    label: Build Artifacts
+    agents:
+      provider: "gcp"
```

Comment:
Maybe for the build step, but since we require to run a docker container in the publish step itself, I understand that is just easier to use the gcp provider, does it make sense?